src/installation/when-to-use-handlebars.md

```diff
@@ -18,7 +10,26 @@ HTML-pages, e-mails or markdown files.
 
 It has **no** built-in support for **event-handling**, accessing **backend-services** or incremental **DOM updates**.
 
-If you want to build a single-page application and you want to handle user-input, you should probably look for a
-framework like Angular, Ember, Ractive, React or Vue.js.
+If you want to build a **single-page application** and you want to handle user-input, you should probably look for a
+framework like 
+
+- [Angular](https://angular.io/)
+- [Aurelia](https://aurelia.io/)
+- [Ember](https://emberjs.com/)
+- [Inferno](https://infernojs.org/)
+- [Mithril](https://mithril.js.org/)
+- [Svelte](https://svelte.dev/)
+- [Ractive](https://ractive.js.org/)
+- [React](https://reactjs.org/)
+- [Vue](https://vuejs.org/)
 
 :::
+
+# PRO HANDLEBARS
```

Comment:
Could you remove the capitalization and use regular text like in the other headers.